test/JDBC/upgrade/14_5/SYSTEM_USER_dep-vu-verify.mix

```diff
@@ -0,0 +1,9 @@
+-- tsql
```

Comment:
These verification and cleanup files should be inside `verification_cleanup` directory.

Reply:
Those files are not in neither the verification or cleanup folder right now in GitHub. Do you think we need to move them to verification_cleanup folder?

Reply:
Oh ok, looks like these files have been added to wrong place from the beginning itself, but the issue is that these tests won't get executed if we keep them here. I think you can keep these here and ask author to move those files in correct folder.